app/templates/admin/cceMinor.html

```diff
@@ -60,70 +60,146 @@ <h2>CCE Minor Progress</h2>
+          Email All
+        </button>
+        <ul class="dropdown-menu" aria-labelledby="emailAllParticipantsDropdown">
+          <li><a class="dropdown-item" href="#" id="emailAllInterested">Interested</a></li>
+          <li><a class="dropdown-item" href="#" id="emailAllDeclared">Declared</a></li>
+          <li><hr class="dropdown-divider"></li>
+          <li><a class="dropdown-item" href="#" id="emailAll">Both Groups</a></li>
+        </ul>
+      </div>
     </div>
 </div>
-<table class="display" id="interestedStudentsTable">
-    <thead>
-        <tr>
-            <th>Name</th>
-            <th> Remove Student</th>
-        </tr>
-    </thead>
-    <tbody>
-        {% for student in interestedStudentsList %}
-            <tr>
-                <td style = "width: 82%">
-                    <a href="/profile/{{student.username}}/cceMinor" target="_blank">{{student.firstName}} {{student.lastName}}</a>
-                </td>
-                <td style="text-align:center">
-                  <button
-                    type="button"
-                    class="remove_interested_student btn btn-danger btn-sm"
-                    id = "{{student.username}}"
-                    aria-label="remove interested student">
-                    <span class="bi bi-trash h5 align-middle"></span>
-                  </button>
-                </td>
-            </tr>
-        {% endfor %}
-        <input id="interestedStudentEmails" hidden value="{{interestedStudentEmailString}}"/>
-    </tbody>
-</table>
+
+
+
+
+{% macro renderStudents(tabData, defaultActiveTab='interested') %}
+<div class="row">
+  <ul class="nav nav-tabs" id="studentTabs" role="tablist">
+    {% for tab in tabData %}
+        <li class="nav-item" role="presentation">
+          <button class="nav-link {% if tab.id == defaultActiveTab %}active{% endif %}" 
+                  id="{{ tab.id }}-tab" 
+                  data-bs-toggle="tab" 
+                  data-bs-target="#{{ tab.id }}" 
+                  type="button" 
+                  role="tab" 
+                  aria-controls="{{ tab.id }}" 
+                  aria-selected="{% if tab.id == defaultActiveTab %}true{% else %}false{% endif %}">
+            {{ tab.label }}
+          </button>
+        </li>
+    {% endfor %}
+  </ul>
+  
+  <div class="tab-content mt-3" id="studentTabsContent">
+    <form id="updateMinorInterestForm" class="d-none" method="POST"></form>
+    
+    {% for tab in tabData %}
+        <div class="tab-pane fade {% if tab.id == defaultActiveTab %}show active{% endif %}" 
+             id="{{ tab.id }}" 
+             role="tabpanel" 
+             aria-labelledby="{{ tab.id }}-tab">
+          <table class="display" id="{{ tab.id }}StudentsTable">
+            <thead>
+              <tr>
+                <th class="w-50">Name</th>
+                <th class="text-center"></th>
```

Comment:
I think it could be good to put both the buttons in the same column, personally.